app/controllers/products_controller.rb

```diff
@@ -35,40 +36,28 @@ def show
 
   def new
     default_img = "https://lh3.googleusercontent.com/pw/ACtC-3eMhEM2kaTc-RlRyVudYKP08KOdRO6QbvXTc_PkmzKzXTIkCqRDIa06GMT1FaJr-lDgIcjmnR5hEEFOCYf4YUDKfozbnOhaOw02IpXMOTr2LW4L2S2PXJfedaWYHq6uTewLUuufgMD0VBs_xdtE7FUy=w1350-h900-no?authuser=0"
-    # if params[:merchant_id]
-    # This is the nested route, /merchant/:merchant_id/products/new
-      @categories = Category.all.order("name DESC")
-      @product = Product.new
-      @product.img_url = default_img
-      @product.price = "0.00"
-      @product.inventory = 0
-    # else
-    # # non-merchants cannot create products
-    #   flash[:error] = "You must login to do that"
-    #   redirect_to products_path
-    # end
+
+    @categories = Category.all.order("name DESC")
+    @product = Product.new
+    @product.img_url = default_img
+    @product.price = "0.00"
+    @product.inventory = 0
   end
 
```

Comment:
Should @product.price be a float instead of a string? Though I guess this is to correctly display a form field placeholder, so it should be fine🙂

Reply:
Oop
That's a good point
I'm trying to remember why I did that

Reply:
Ah! I wanted it to show up as 0.00 on the form, and there isn't a view helper for that yet. I'll change it to a float while the view helper is being made. Danke!